website/content/docs/concepts/domain-model/credential-stores.mdx

```diff
@@ -161,14 +169,16 @@ $ curl https://boundaryproject.io/data/vault/boundary-controller-policy.hcl -O -
 $ vault policy write boundary-controller boundary-controller-policy.hcl
 ```
 
+### Static Credential Store
+
+A Static credential store allows for a user-supplied credentials managed by Boundary. Credentials are encrypted and stored directly in Boundary. Currently, the static credential store can hold credentials of type `username_password`.
```

Comment:
I'm not sure I understand this sentence. Is it trying to say: A static credential store allows user-supplied credentials  to be managed by Boundary?